src/main/java/com/alyrow/gdx/particle/ParticleType.java

```diff
@@ -1,11 +1,62 @@
 package com.alyrow.gdx.particle;
 
+import com.alyrow.gdx.particle.texture.ParticleTexture;
+import com.badlogic.gdx.graphics.Texture;
+
 /**
  * @author alyrow
  * Define particles type...
  */
-public class ParticleType {
-    public static int HALO = 0;
-    public static int TEXTURE = 1;
-    public static int NOTHING = 2;
+public enum ParticleType {
+    CIRCLE("circle.png"),
+    DISCORD("discord.png"),
+    GRILL("grill.png"),
+    HALO("halo.png"),
+    SQUARE("square.png"),
+    STAR("star.png"),
+    TRIANGLE("triangle.png"),
+
+    TEXTURE(),
+    NOTHING();
```

Comment:
Modification to texture

Reply:
Okay too
https://github.com/alyrow/box2d-light-particle/commit/a6a176eb1898c9fef3f38168ed68ed7699223ff7